ostelco-ios-client/ViewControllers/Home/ApplePayViewController.swift

```diff
@@ -61,18 +61,9 @@ class ApplePayViewController: UIViewController, ApplePayDelegate {
 
     func showOhNo(type: OhNoViewController.IssueType) {
         let ohNo = OhNoViewController.fromStoryboard(type: type)
+        ohNo.buttonTitle = "Ok I understand"
```

Comment:
I would recommend putting this in the `IssueType`'s property for the title. If it doesn't have a property for that yet, add one 😄